src/openrct2/config/Config.cpp

```diff
@@ -336,6 +336,11 @@ namespace Config
             auto model = &gConfigNetwork;
             model->player_name = reader->GetCString("player_name", "Player");
             model->default_port = reader->GetSint32("default_port", NETWORK_DEFAULT_PORT);
+
+            model->listen_address = reader->GetCString("listen_address", "");
+            if (strlen(model->listen_address) == 0)
+                model->listen_address = nullptr;
```

Comment:
This would cause an empty string (not a nullptr) once the config gets saved and reloaded. That's why I did this check.